st2client/st2client/commands/action.py

```diff
@@ -41,10 +41,24 @@
 
 LIVEACTION_STATUS_REQUESTED = 'requested'
 LIVEACTION_STATUS_SCHEDULED = 'scheduled'
+LIVEACTION_STATUS_DELAYED = 'delayed'
 LIVEACTION_STATUS_RUNNING = 'running'
+LIVEACTION_STATUS_SUCCEEDED = 'succeeded'
+LIVEACTION_STATUS_FAILED = 'failed'
+LIVEACTION_STATUS_TIMED_OUT = 'timeout'
+LIVEACTION_STATUS_ABANDONED = 'abandoned'
 LIVEACTION_STATUS_CANCELING = 'canceling'
 LIVEACTION_STATUS_CANCELED = 'canceled'
 
+
+LIVEACTION_COMPLETED_STATES = [
```

Comment:
Needed to copy this over since we still can't have st2client to depend on st2common.

Packages will make that easier though, we just need to make sure we also publish st2common to PyPi.